### PR TITLE
➕ add typescript

### PR DESCRIPTION
**Pull request by RonjaMeinecke**
https://nextjs.org/learn/excel/typescript/create-tsconfig
• add tsconfig.json via `https://nextjs.org/learn/excel/typescript/create-tsconfig`
• `npm run dev` to install typescript dev dependencies in tsconfig.json

### CODE REVIEW

**Review by m-wiemers — CHANGES_REQUESTED** [review]
Rename the files - then it`s looks better ;-) 

@m-wiemers not sure what you mean 🤔

**Review by christophvogel — APPROVED** [review]
GJ 👍

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "jsx": "preserve"
+  },
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx"
+  ],
+  "exclude": [
+    "node_modules"
+  ]
```
Comment (phlgr) [inline tsconfig.json]: These need to be prettified.

**Review by phlgr — CHANGES_REQUESTED** [review]

**Review by phlgr — APPROVED** [review]
🚀

**Pull request merged**